tests/integration/targets/wafv2/meta/main.yml

```diff
@@ -0,0 +1,4 @@
+dependencies:
```

Comment:
You'll also need to add 
```
      vars:
        ansible_python_interpreter: "{{ botocore_virtualenv_interpreter }}"
```

to the include in tasks/main.yml
```
 - include_tasks: test_webacl.yml
   vars:
      ansible_python_interpreter: "{{ botocore_virtualenv_interpreter }}"
```

Reply:
Done now, hope I din't overdo it ;)